frontend/src/general/__tests__/SnackbarMessage.test.jsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  it,
+  jest
+} from "@jest/globals";
+import { Snackbar, SnackbarContent, Box } from "@material-ui/core";
+import { CheckCircle, Error } from "@material-ui/icons";
+import SnackbarMessage from "../SnackbarMessage";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<SnackbarMessage /> Class Component", () => {
```

Comment:
i see the exported `snackbar()` isn't tested in here. i know it gets tested in other test files but sonarcloud still shows the line as uncovered. is it not possible to just call `snackbar()` in here to check that it's calling `openSnackbar()` with the correct info?

Reply:
I believe it's more complicated than that and I personally made the decision to not spend additional time investigating how to test it.